fusor-ember-cli/app/controllers/openstack/overcloud.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+import DeploymentControllerMixin from "../../mixins/deployment-controller-mixin";
+import NeedsDeploymentMixin from "../../mixins/needs-deployment-mixin";
+
+export default Ember.Controller.extend(DeploymentControllerMixin, NeedsDeploymentMixin, {
+
+  isCloudForms: Ember.computed.alias("deploymentController.isCloudForms"),
+
+  nextStepRouteNameOvercloud: Ember.computed('isCloudForms', function() {
+    if (this.get('isCloudForms')) {
+      return 'cloudforms';
+    } else {
+      return 'subscriptions';
+    }
+  }),
+
+  isValidPrivateNetworkRange: Ember.computed('model.openstack_overcloud_private_net', function() {
+      // TODO
+      return true;
```

Comment:
This PR doesn't validate if network range is valid or it. This will be in a separate PR, but he's the placeholder for it.